lms/templates/student_account/form_field.underscore

```diff
@@ -33,6 +33,8 @@
             <%  });
             } %>
             <% if ( required ) { %> aria-required="true" required<% } %>
+            <% if ( typeof disabled !== 'undefined' && disabled ) { %> disabled<% } %>
+
```

Comment:
What is disabled here? IF you are disabling select/dropdown, a learner can easily edit out HTML and enable it. The course id passed to view for new flow should not be dependent on the dropdown.

Reply:
Using a workaround here:
```
        course_id = data['course']
        if course_id not in request.META.get('HTTP_REFERER'):
            return HttpResponseBadRequest('Invalid Course ID provided.')
```